janus_core/stats.py

```diff
@@ -0,0 +1,70 @@
+"""
+Module that reads the md stats output timeseries.
+"""
+
+import re
+
+from numpy import genfromtxt
+
+from janus_core.janus_types import PathLike
+
+
+class Stats:
+    """
+    Configure shared molecular dynamics simulation options.
+
+    Parameters
+    ----------
+    source : PathLike
+        File that contains the stats of a molecular dynamics simulation.
+    """
+
+    def __init__(self, source: PathLike = None) -> None:
```

Comment:
As it is, this should probably be `Optional[PathLike]`, but I'm not sure I see the benefit in it being optional given that the other functions both require a file to have any utility?